### PR TITLE
Hide and log unknown events

### PR DESCRIPTION
**Pull request by cvalarida**
There's not much by way of screenshots, but I did test with mock data to ensure it's working properly.

### CODE REVIEW

File: src/js/claims-status/utils/appeals-v2-helpers.jsx [hunk 1]
```diff
@@ -650,10 +650,12 @@ export function getEventContent(event) {
         description: '',
       };
     default:
-      return {
-        title: 'Unknown event',
-        description: '',
-      };
+      Raven.captureMessage('appeals-unknown-event', {
+        extra: {
+          eventType: event.type
```
Comment (cehsu) [inline src/js/claims-status/utils/appeals-v2-helpers.jsx]: Would any other information about the event be helpful to track?

**Review by cehsu — APPROVED** [review]
LGTM. Had a question about what else (if anything), might also be of interest to track when logging the error.

File: src/js/claims-status/utils/appeals-v2-helpers.jsx (hunk 1 above)
Comment (cvalarida) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: :thinking: I don't think so, but @cmgiven might have an idea. What do you think, Chris?

Comment (cmgiven) [reply src/js/claims-status/utils/appeals-v2-helpers.jsx]: I think that would do it. This helpfully flags a mismatch between the API and the front-end. Knowing is most of the battle, and we can quickly figure out the rest once we know there's an issue.

**Pull request merged**